src/main/java/seedu/contax/logic/commands/RangeCommand.java

```diff
@@ -34,7 +33,9 @@
             + PREFIX_RANGE_TO + "TO *\n"
             + "Example: `" + COMMAND_WORD + " edit "
             + PREFIX_PHONE + "12345678 "
-            + PREFIX_ADDRESS + "New Address `";
+            + PREFIX_ADDRESS + "New Address "
+            + PREFIX_RANGE_FROM + "1 "
+            + PREFIX_RANGE_TO + "3 `\n";
```

Comment:
There is no need for the trailing breakline.

Reply:
thx removed